### PR TITLE
Feature/rh 180_email_and_tel_clickable

### LINKED ISSUE #180: email & tel clickable
As a user I want to click the email/telephone number in a person's detail page to directly contact the person.

clicking the email address allows me to write an email to the person, telephone number to take a call

DoD
- [ ]  Tests
- [ ]  Documentation
- [ ]  CI Build passing
- [ ]  Peer Review
- [ ]  Approval from PO

### PR DESCRIPTION
**Pull request by rfrx**
* closes #180 
* now users can click on people's phone numbers resp. email addresses on people show and index page and can directly call resp. email the person

### CODE REVIEW

File: app/javascript/packs/ajax.js [hunk 1]
```diff
@@ -18,6 +18,12 @@ global.ajaxCall = (
     $("#browse-outlet a").each(function () {
       const link = $(this);
       const href = link.attr("href");
+      if (href.startsWith("mailto:")){
+        return;
+      }
+      if (href.startsWith("tel:")){
+        return;
+      }
```
Comment (TheGrayStone) [inline app/javascript/packs/ajax.js:26]: Maybe shrinking it down a bit would be better:

```suggestion
      if (href.startsWith("mailto:") || href.startsWith("tel:")){
        return;
      }
```

Or even something like:
```suggestion
      if (href.match(/(mailto|tel):(.*)/)) {
        return;
      }
```

File: app/javascript/packs/ajax.js [hunk 2]
```diff
@@ -30,7 +36,7 @@ global.ajaxCall = (
   /** Special handling of all /map/route paths so that instead of loading another page,
    * the route navigation screen is shown on in the browse outlet so that a navigation can be started.
    */
-  if (target === "/route") {
+   if (target === "/route") {
```
Comment (TheGrayStone) [inline app/javascript/packs/ajax.js:39]: I suspect this is a space too much 😅 
```suggestion
  if (target === "/route") {
```

File: app/views/people/index.html.erb [hunk 3]
```diff
@@ -19,8 +19,8 @@
       <tr>
         <td><%= person.first_name %></td>
         <td><%= person.last_name %></td>
-        <td><%= person.email %></td>
-        <td><%= person.phone_number %></td>
+        <td><%= mail_to person.email %></td>
+        <td><%= tel_to person.phone_number %></td>
```
Comment (TheGrayStone) [inline app/views/people/index.html.erb:23]: I like the helpers 👍

**Review by TheGrayStone — CHANGES_REQUESTED** [review]
Looks good 👍 

I've just some little change requests.

(Also: It seems you still have to deal with code factor)

**Review by philbow61 — COMMENTED** [review]
Good Job 👍

**Review by TheGrayStone — APPROVED** [review]

**Pull request merged**